terra_node_bot.py

```diff
@@ -0,0 +1,584 @@
+    return show_home_menu_new_msg(context=context, chat_id=update.effective_chat.id)
+
+
+def node_details(update, context):
+    """
+    Show Detail Menu for the selected address
+    """
+
+    query = update.callback_query
+
+    address = query.data.split("-")[1]
+    context.user_data['selected_node_address'] = address
+
+    return show_detail_menu(update=update, context=context)
+
+
+def confirm_node_deletion(update, context):
+    """
+    Ask user if he is sure to remove the address
+    """
+
+    address = context.user_data['selected_node_address']
+
+    keyboard = [[
+        InlineKeyboardButton('YES ✅', callback_data='delete_node'),
+        InlineKeyboardButton('NO ❌', callback_data='show_detail_menu')
+    ]]
+    text = '⚠️ Do you really want to remove the address from your monitoring list? ⚠️\n*' + address + '*'
+
+    query = update.callback_query
+    query.edit_message_text(text, parse_mode='markdown', reply_markup=InlineKeyboardMarkup(keyboard))
+
+
+def delete_node(update, context):
+    """
+    Remove address vom the monitoring list
+    """
+
+    query = update.callback_query
+    address = context.user_data['selected_node_address']
+
+    del context.user_data['nodes'][address]
+
+    text = "❌ Node address got deleted! ❌\n" + address
+    query.answer(text)
+    query.edit_message_text(text)
+    show_home_menu_new_msg(context=context, chat_id=update.effective_chat.id)
+
+
+"""
+######################################################################################################################################################
+Helpers
+######################################################################################################################################################
+"""
+
+
+def get_home_menu_buttons(user_data):
+    """
+    Return Keyboard buttons for the home menu
+    """
+
+    keyboard = [[]]
+
+    for address in user_data['nodes'].keys():
+        keyboard.append([InlineKeyboardButton(address, callback_data='node_details-' + address)])
+
+    keyboard.append([InlineKeyboardButton('Add Node', callback_data='add_node')])
+
+    return keyboard
+
+
+def show_detail_menu(update, context):
+    """
+    Show detail buttons for selected address
+    """
+
+    query = update.callback_query
+    address = context.user_data['selected_node_address']
+
+    text = 'Node: *' + address + '*\n' + \
+           'Status: *' + NODE_STATUSES[context.user_data['nodes'][address]['status']] + '*\n' + \
+           'Jailed: *' + str(context.user_data['nodes'][address]['jailed']) + '*\n\n'
+
+    text += "What do you want to do with that Node?"
+
+    keyboard = [[
+        InlineKeyboardButton('Delete Node', callback_data='confirm_node_deletion'),
+        InlineKeyboardButton('<< Back', callback_data='home')
+        ]]
+
+    # Modify message
+    query.edit_message_text(text, parse_mode='markdown', reply_markup=InlineKeyboardMarkup(keyboard))
+
+
+def expect(want, user_data):
```

Comment:
Why use a function for that and not directly just set the value? `expect` sounds something like `assert`, as if you are doing assertion checking.